.gitignore

```diff
@@ -10,3 +10,5 @@ pack
 coverage
 .nyc_output
 .LAST_BUILD
+*.swp
+./examples/cdk-examples-typescript/hello-cdk-ecs/cdk.json
```

Comment:
Put this in a .gitignore file under `./examples/cdk-examples-typescript/hello-cdk-ecs/`